src/plugins/calendar/calendar.js

```diff
@@ -28,25 +28,27 @@ var i18nText,
 			goToMonth: i18n( "cal-goToMnth" ),
 			dayNames: i18n( "days" ),
 			currDay: i18n( "currDay" ),
-			format: i18n( "cal-format" )
+			format: i18n( "cal-format" ),
+			calendar: i18n( "cal" )
 		};
 
 		textWeekDayNames = i18nText.dayNames;
 		textMonthNames = i18nText.monthNames;
 
-		$calBase = $( "<div class='wb-clndr' role='application'>" +
+		$calBase = $( "<div class='wb-clndr' role='application' aria-label='" + i18nText.calendar + "'>" +
 			"<div class='cal-nav'>" +
+				"<span class='wb-inv current-month' aria-live='polite'></span>" + // Added for screen-readers
```

Comment:
Should we change the aria label on `.wb-clndr` instead or at least change it also and keep both synchronized after the first interaction?

If possible, @Garneauma can you test with a SR: Ideally we will have an heading, figcaption, caption... something to identify the calendar title. Here I know it's the aria-label but I don't know how will be experience if `aria-live="polite"` is set on the `.wb-clndr` element when we change the aria-label and all the inner controls. I do think we don't necessary want the inner control changes to be announced but to announce the whole `role=application` change.

Reply:
Adding aria-live to the .wb-clndr element makes the SR announce too much when clicking on the previous or next arrows. This results in the SR announcing something like: "March Month: March Monday, March 1, 2013 Tuesday, March 2, 2013, ..." and is very annoying. By defining a specific element to keep track, it is much more understandable for a SR user to understand what their action did. However, I realized that only announcing the Month might not be enough. I have just added the year too. So it now reads something like: "March 2022".